main.js

```diff
@@ -1,12 +1,49 @@
 'use strict';
 
-const logger = require('ft-next-logger');
+const logger = require('ft-next-logger').logger;
+const fetchres = require('fetchres');
+const raven = require('raven');
+let ravenMiddleware;
+
+function sendErrorDev (err, req, res) {
+	if (err.name === fetchres.ReadTimeoutError.name) {
+		logger.error('Dependency timeout -', err);
```

Comment:
Absolutely, just check Winston isn't doing it behind the scenes. Wanted to make logger a bit more cleverer around error object generally, as those are the things that are usually passed to it